interface/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@brave/swap-interface",
-  "version": "0.12.4",
+  "version": "0.12.5",
```

Comment:
Please run `npm install` in this directory and also in `sites/mock` to update the versions in the package-lock.json files.